dotnet-authserver/tests/TeacherIdentity.AuthServer.EndToEndTests/HostFixture.cs

```diff
@@ -31,22 +30,19 @@ public class HostFixture : IAsyncLifetime
     private bool _disposed = false;
 
     private readonly List<string> _capturedAccessTokens = new();
-    private readonly List<(string Email, string Pin)> _capturedEmailConfirmationPins = new();
+
+    public static string UserVerificationPin { get; } = "12345";
```

Comment:
This is set statically so that extension methods (e.g. on `PageExtensions`) could access it directly without having to accept a parameter.